spring-cloud-huawei-discovery/src/main/java/com/huaweicloud/servicecomb/discovery/client/model/ServiceCombServiceInstance.java

```diff
@@ -88,8 +89,8 @@ public int getPort() {
 
   @Override
   public boolean isSecure() {
-    // TODO: add secure implementation
-    return false;
+    URIEndpointObject uriEndpointObject = new URIEndpointObject(this.microserviceInstance.getEndpoints().get(0));
```

Comment:
Initialize a URIEndpointObject in constructor. And implement other methods, like getPort, getHost, getSchema by read URIEndpointObject